apps/server/src/helpers/email/newUser.ts

```diff
@@ -0,0 +1,242 @@
+                                </td>
+                              </tr>
+                              <tr>
+                                <td align="left" style="font-size:0px;padding:8px;word-break:break-word;">
+                                  <div style="font-family:'Helvetica', 'Arial', sans-serif;font-size:40px;line-height:1.25;text-align:left;color:#ffffff;"><strong>Your account has been created!</strong></div>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+                <!--[if mso | IE]></td></tr></table><![endif]-->
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <!--[if mso | IE]></td></tr></table><table align="center" border="0" cellpadding="0" cellspacing="0" class="md-rounded-bottom" style="width:600px;" width="600" bgcolor="#ffffff" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]-->
+      <div class="md-rounded-bottom" style="background:#ffffff;background-color:#ffffff;margin:0px auto;max-width:600px;">
+        <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background:#ffffff;background-color:#ffffff;width:100%;">
+          <tbody>
+            <tr>
+              <td style="direction:ltr;font-size:0px;padding:30px;text-align:center;">
+                <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:540px;" ><![endif]-->
+                <div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
+                  <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%">
+                    <tbody>
+                      <tr>
+                        <td style="vertical-align:top;padding:0;">
+                          <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="" width="100%">
+                            <tbody>
+                              <tr>
+                                <td align="left" style="font-size:0px;padding:8px;word-break:break-word;">
+                                  <div style="font-family:'Helvetica', 'Arial', sans-serif;font-size:20px;line-height:1.5;text-align:left;color:#000000;">
+                                  Dear ${person.name},<br /><br />
+                                  Your account has been updated on the Raise platform. <br /><br ?>
+                                  You should be able to login at <a href="https://www.joinraise.org/admin/" target="_blanck" >www.joinraise.org/admin/</a>, using your account ${person.email}. <br /><br />
```

Comment:
```suggestion
                                  You should be able to login at <a href="https://www.joinraise.org/admin/" target="_blank" >www.joinraise.org/admin/</a>, using your account ${person.email}. <br /><br />
```

Also, I think we it'd be good to extract the base URL to `env` here. Basically can copy out CUSTOM_RAISE_DOMAIN (and probably CUSTOM_MWA_DOMAIN) from the web env into the server env, then reference that here. The benefit of doing this is it makes everything work cross-environment nicely (having most things pointing in the right places, and some things that always point at prod is very confusing and often results in making big mistakes e.g. thinking you're going to the dev site and then deleting a load of prod data is not good!), and changing domain in future is made much easier.

Reply:
Sure will do! Was wondering if I should a /admin tag to the end of the local host address? 

Also, how should I ensure that this will update to the correct value when not in a production environment? I assume that local.ts within env is the prod copy, and env is the real world copy?

Reply:
Yes, I think there should be a `/admin` at the end - added to the base URL (not in env)

> Also, how should I ensure that this will update to the correct value when not in a production environment? I assume that local.ts within env is the prod copy, and env is the real world copy?

`local.ts` is the localhost version. I have the dev and prod copies and will update them - on reflection these should be more accessible, but I haven't figured that system out yet.